lib/Api.js

```diff
@@ -37,6 +37,7 @@ var translatedName = require("./nameTranslation");
 var tokens = require("./tokens");
 var syntax_dependencies = require("./syntax_dependencies");
 var transliteration = require("./transliteration");
+var topics = require("./topics.js");
```

Comment:
why `.js` only for this ep?

Reply:
Typed too fast.  It works but is unnecessary